Promise/Promise+Extras.swift

```diff
@@ -50,8 +60,14 @@ public enum Promises {
         })
     }
 
-    /// Fulfills or rejects with the first promise that completes
-    /// (as opposed to waiting for all of them, like `.all()` does).
+    /// Fulfills or rejects with the first promise that completes.
+    /// - Parameter promises: Promises to race
+    /// - Returns: A `Promise<T>` that will be fulfilled or rejected with
+    ///            the first of `promises` that completes.
+    ///
+    /// Note that `race()` is different to `all()` because `race()` will
+    /// blindly take the first promise to complete and returns that promise,
+    /// where `all()` will wait for every promise to complete and return all of them.
```

Comment:
```suggestion
    /// Note that `race()` is different to `all()` because `race()` will
    /// take the first promise to complete and returns that promise's results,
    /// where `all()` will wait for every promise to complete and return all of them.
```

Reply:
CML: `results` → `result`

Reply:
Fixed in c9bc94b448931d667b510320c0b65e9096386dd2